assets/src/edit-story/output/page.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import StoryPropTypes from '../types';
+import { PAGE_WIDTH, PAGE_HEIGHT } from '../constants';
+import OutputElement from './element';
+
+function OutputPage( { page } ) {
+	const { id } = page;
+	// Aspect-ratio constraints.
+	const aspectRatioStyles = {
+		width: `calc(100 * var(--story-page-vw))`, // 100vw
+		height: `calc(100 * ${ PAGE_HEIGHT / PAGE_WIDTH } * var(--story-page-vw))`, // 16/9 * 100vw
+		maxHeight: `calc(100 * var(--story-page-vh))`, // 100vh
+		maxWidth: `calc(100 * ${ PAGE_WIDTH / PAGE_HEIGHT } * var(--story-page-vh))`, // 9/16 * 100vh
+		// todo@: this expression uses CSS `min()`, which is still very sparsely
```

Comment:
Should this (the `min()`) be addressed already now? It looks like it's not supported in Firefox at all which would likely mean the font not being displayed. Thoughts?

Reply:
I think this will just have to be wrong for now until we get some assistance from the Runtime team to make it cross-browser.